pkg/info/types.go

```diff
@@ -0,0 +1,28 @@
+package info
+
+const (
+	SocketAddress = "/var/run/numaflow/info.sock"
+)
+
+type Protocol string
+
+const (
+	UDS Protocol = "uds"
+	TCP Protocol = "tcp"
+)
+
+type Language string
+
+const (
+	Go     Language = "go"
+	Python Language = "python"
+	Java   Language = "java"
```

Comment:
why do we need to track each language on numaflow-go?

Reply:
Our sdk versions are not consistent. We should have a mapping in numaflow code to track all the versions for compatibility check.